test/command/cmd.go

```diff
@@ -60,6 +61,7 @@ func init() {
 	supernodePath = fp.Join(binDir, "supernode")
 }
 
+// checkExist checks the path s whether is existing.
```

Comment:
The `s` is weird. Could you please update the param name to path or something else? THX.

Reply:
OK  My mistake